data/animeseries

```diff
@@ -85,3 +85,7 @@ Your lie in April
 The Rising of the Shield Hero
 Assassination Classroom
 ERASED
+Fate/stay night
+Fate/Grand Order
+Fate/Apocrypha
```

Comment:
Common, if you're going to do Fate you need to include Fate/Zero.

Reply:
Thats valid, I will do so.

Reply:
This has been resolved.